src/main/java/org/altbeacon/beacon/service/RunningAverageRssiFilter.java

```diff
@@ -0,0 +1,78 @@
+package org.altbeacon.beacon.service;
+
+import org.altbeacon.beacon.logging.LogManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Iterator;
+
+/**
+ * Created by AS on 17.02.2015.
+ */
+public class RunningAverageRssiFilter implements RssiFilter {
```

Comment:
I know this is not your code, but now that it is a separate class, it would be great to have a `RunningAverageRssiFilterTest` to add test coverage for this, too